draft-ietf-tls-esni.md

```diff
@@ -819,13 +827,20 @@ Clients SHOULD NOT attempt to repair the connection in this case.
 
 # Security Considerations
 
-## Security and Privacy Goals
+## Security and Privacy Goals {#goals}
+
+ECH considers two types of attackers: passive and active. Passive attackers can
+read packets from the network. They cannot perform any sort of active behavior
+such as probing servers or querying DNS. A middlebox that filters based on
+plaintext packet contents is one example of a passive attacker. In contrast,
+active attackers can write packets into the network for malicious purposes, such
+as interfering with existing connections, probing servers, and querying DNS. A
+censor is one example of an active attacker.
```

Comment:
A more important example of an active attacker is the one that tries to break the primary security goal of ECH. A cesnor is an example of a "don't stick out" attacker. I suggest replacing the last line with something like this: 

> In short, an active attacker corresponds to the conventional threat model for TLS ({{RFC8446}}, Section ??).

Reply:
I'll add that, instead of replace the censor example.